.setup/data/courses/development.yml

```diff
@@ -591,17 +591,17 @@ gradeables:
             gcm_points: -5
     g_ta_view_start_date: -2 days
     eg_submission_open_date: -1 days
-    eg_submission_due_date: +1000 days
-    g_grade_start_date: +1005 days
-    g_grade_due_date: +1010 days
-    g_grade_released_date: +1010 days
-    eg_grade_inquiry_start_date: +1011 days
-    eg_grade_inquiry_due_date: +1015 days
+    eg_submission_due_date: +25 days
+    g_grade_start_date: +30 days
+    g_grade_due_date: +32 days
+    g_grade_released_date: +32 days
+    eg_grade_inquiry_start_date: +33 days
+    eg_grade_inquiry_due_date: +38 days
     eg_has_due_date: true
     eg_has_release_date: true
     eg_max_random_submissions: 0
 
-  - gradeable_config: python_multipart_static_analysis
+  - gradeable_config: python_linehighlight
```

Comment:
Why are the gradeable_configs getting changed? Is this necessary?

Reply:
One of them was out of order, so I put it in the correct order. Due to this, all subsequent configs look like they were changed